frontend/packages/dev-console/src/components/edit-application/__tests__/edit-application-data.ts

```diff
@@ -249,6 +249,7 @@ export const appResources: AppResources = {
         annotations: {
           'app.openshift.io/vcs-ref': 'master',
           'app.openshift.io/vcs-uri': 'https://github.com/divyanshiGupta/nationalparks-py',
+          jarFileName: 'demo-app.jar',
```

Comment:
all three of these annotations will be not be used together right?

Reply:
No, jarFileName will be used to form Jar initial values and the other is for git import.